PowerFGT/Public/Connection.ps1

```diff
@@ -76,6 +82,8 @@ function Connect-FGT {
         [Parameter(Mandatory = $false)]
         [SecureString]$Password,
         [Parameter(Mandatory = $false)]
+        [string]$ApiToken,
```

Comment:
May be use a ParameterSet for don't use on the same time ApiToken and other method...

Reply:
ParameterSet added